### PR TITLE
Move doc and config files in repo of analytics-zoo-doc to analytics-zoo

### PR DESCRIPTION
**Pull request by helenlly**
1) create a new folder "doc" for build  docs in readthedoc
2) move doc and config files in repo of analytics-zoo-doc to analytics-zoo
3) test build works, see the link https://helentestdoc.readthedocs.io/en/latest/

PS: after merged, 
1) will update the project "https://readthedocs.org/projects/analytics-zoo/" 
2) use repo intel-analytics/analytics-zoo  instead of analytics-zoo/analytics-zoo-doc
3) analytics-zoo/analytics-zoo-doc can be deleted.

### CODE REVIEW

**Comment by jason-dai** [comment]
1) Remove files that are not needed
2) Update the index.rst file
3) The project name should be "Analytics Zoo", not "analytics-zoo"
4) Rename `doc` to `readthedocs`, and `source\readthedocs` to source\doc`

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name = "analytics-zoo"
```
Comment (jason-dai) [inline pyproject.toml:1]: ""Analytics Zoo"

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+version = "0.1"
+description = "A unified Data Analytics and AI platform for distributed TensorFlow, Keras and PyTorch on Apache Spark/Flink & Ray"
```
Comment (jason-dai) [inline pyproject.toml:3]: An open source Big Data AI platform (for distributed TensorFlow, PyTorch and Keras on Apache Spark & Ray)

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+license = "Apache"
```
Comment (jason-dai) [inline pyproject.toml:4]: Apache-2.0 License

Comment (helenlly) [reply pyproject.toml:4]: receive above comments, will cancel this one and re-submit a PR,

**Pull request closed without merging**